rotkehlchen/tests/api/test_current_assets_price.py

```diff
@@ -282,10 +282,15 @@ def test_remove_manual_current_price(rotkehlchen_api_server):
     assert (A_BTC, A_ETH) not in Inquirer()._cached_current_price
 
 
+@pytest.mark.vcr()
 @pytest.mark.parametrize('should_mock_current_price_queries', [False])
 def test_manual_current_prices_loop(inquirer):
     """Check that if we got a loop of manual current prices
```

Comment:
For this test did you also check that the warnings are correct?

For example from this output from https://github.com/rotki/rotki/actions/runs/4435118738/jobs/7781949206

```
FAILED rotkehlchen/tests/api/test_current_assets_price.py::test_manual_current_prices_loop[False] - AssertionError: assert 2 == 1
 +  where 2 = len(['Was not able to find price from ETH(Ethereum) to EUR(Euro) since your manual latest prices form a loop. For now, oth...(United States Dollar) to EUR(Euro) since your manual latest prices form a loop. For now, other oracles will be used.'])
```

It looks like there are 2 warnings about loops, which I think is not correct. I think there should be one warning about loops and one warning about inability to query price.

Reply:
So the reason for the two warnings is:
We call 
```python
price = inquirer.find_price(
        from_asset=A_ETH.resolve_to_asset_with_oracles(),
        to_asset=A_EUR.resolve_to_asset_with_oracles(),
)
```
with the following oracles = [Manual, Coingecko, Defillama, Cryptocompare]
---> tries the first oracle(manual), enters recursion & adds the first warning...
---> up next coingecko, `RemoteError` occurs, next oracle...
---> tries defillama, it works we get `ETH -> USD`... from the defillama's `query_current_price` method, `Inquirer().find_usd_price()` is called to get `USD->EUR`
--------> calls the first oracle(manual) again, enters recursion & adds the second warning...